pom.xml

```diff
@@ -138,38 +141,17 @@
                 <activeByDefault>true</activeByDefault>
```

Comment:
I was torn as to whether we should even define a profile in the root POM (like HDP-3.1) now that we support only one set of versions.  I left the existing HDP-3.1 profile, which is the default, so whether you build with the profile or without you get the same result.

Reply:
The profile is where all the core "platform" versions are specified.